extensions/guidepost/README.md

```diff
@@ -0,0 +1,40 @@
+# Extension - Guidepost
+## Description
+Used to set guidepost objects onto the canvas.
+The config file can be found at [/run/map/watermark/guideposts.json](../../run/map/watermark/guideposts.json)
+
+## Parameters
+* assetName: the guidepost asset name being called
+* src: path to the asset
+* imageRef: image reference
+* The following parameter can refer to [CanvasRenderingContext2D.drawImage](https://developer.mozilla.org/zh-CN/docs/Web/API/CanvasRenderingContext2D/drawImagehttps://developer.mozilla.org/zh-CN/docs/Web/API/CanvasRenderingContext2D/drawImage) function
+    * srcX: sx. the coordinate X in the image to start from
+    * srcY: sy. the coordinate Y in the image to start from
+    * sWidth: sWidth. the width of image to be used
+    * sHeight: sHeight. the height of image to be used
+    * dWidth: dWidth. the width of image to be zoomed
+    * dHeight: dHeight. the height of image to be zoomed
```

Comment:
Are these parameters optional? If so, maybe add it to the documentation.

Reply:
Indeed, it is optional to CanvasRenderingContext2D.drawImage(), but not to _drawWatermark in MapRenderer, or should I write it as an optional feature?

Reply:
Well ... if the caller does not provide `dWidth` and `dHeight`, then the image does not need to be zoomed. That is, we just use `(sWidth, sHeight)` as `(dWidth, dHeight)`.